addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_ubl_be.py

```diff
@@ -214,7 +214,7 @@ def test_export_import_refund(self):
         self._assert_imported_invoice_from_etree(refund, attachment)
 
     def test_encoding_in_attachment_ubl(self):
-        self._test_encoding_in_attachment('ubl_bis3', 'INV_2017_01_0002_ubl_bis3.xml')
+        self._test_encoding_in_attachment('ubl_bis3', 'INV_2017_00002_ubl_bis3.xml')
```

Comment:
What happened here :thinking:

Reply:
Good question, when running the tests locally, this one failed (with or without this commit). So I don't really know why the nightly build is green on runbot. also, on line 224 of this file, you can see the expected attachment's name is "INV_2017_00002_ubl_bis3.xml" so on of these name has to be wrong.